Source/WebCore/rendering/style/RenderStyle.cpp

```diff
@@ -87,7 +87,7 @@ struct SameSizeAsBorderValue {
 
 static_assert(sizeof(BorderValue) == sizeof(SameSizeAsBorderValue), "BorderValue should not grow");
 
-struct SameSizeAsRenderStyle {
+struct SameSizeAsRenderStyle : public CanMakeCheckedPtr<RenderStyle> {
```

Comment:
```suggestion
struct SameSizeAsRenderStyle : CanMakeCheckedPtr<SameSizeAsRenderStyle> {
```
Should be this as discussed on Slack